validator/packaging/validator.toml.example

```diff
@@ -33,3 +33,10 @@ seeds = ["tcp://127.0.0.1:8801"]
 # A list of peers to attempt to connect to in the format tcp://hostname:port.
 # It defaults to None.
 peers = ["tcp://127.0.0.1:8801"]
+
+# A Curve ZMQ key pair are used to create a secured network based on side-band
```

Comment:
Pair is singular. So you can say "A Curve ZMQ key pair is used" or "Curve ZMQ keys are used".